package/com.unity.formats.usd/Tests/Editor/ExportTests.cs

```diff
@@ -117,5 +117,23 @@ public void ExportMultipleSiblingGameObjectsWithSameName_AllGameObjectsHaveCorre
             }
             Assert.AreEqual(cubes.Length, exportedPrims.Count, "One or more GameObjects don't have a corresponding Prim");
         }
+
+        // TODO: This test will need to be updated in later versions of the package where the UsdGeomPrimvarsAPI is implemented
+        [Test]
+        public void ExportGameObjectWithMesh_STPrimVarInterpolationSetToVarying()
+        {
+            var cube = GameObject.CreatePrimitive(PrimitiveType.Cube);
+            cube.name = "cube";
+
+            ExportHelpers.ExportGameObjects(new GameObject[] { cube }, ExportHelpers.InitForSave(m_USDScenePath), BasisTransformation.SlowAndSafe);
+
+            m_USDScene = Scene.Open(m_USDScenePath);
+            pxr.UsdPrim cubePrim = GetPrim(cube);
+
+            pxr.UsdGeomMesh usdGeomMesh = new pxr.UsdGeomMesh(cubePrim);
+            var stPrimvar = usdGeomMesh.GetPrimvar(new pxr.TfToken("st"));
+            Assert.IsNotNull(stPrimvar, $"Mesh {cube.name} has no 'st' primvar.");
+            Assert.AreEqual(new pxr.TfToken("varying"), stPrimvar.GetInterpolation(), $"st on mesh {cube.name} is not set to varying interpolation.");
```

Comment:
We have the common geometry tokens defined as static TfTokens in UsdGeomTokens. You could use UsdGeomTokens.varying here

Reply:
Oh cool, I didn't know about those, thanks :) Is it worth generating one for st in there too?

Reply:
UsdGeomTokens contains the token values defined by the  various UsdGeom schemas (see https://github.com/Unity-Technologies/usd-unity-sdk/blob/dev/package/com.unity.formats.usd/Runtime/Plugins/x86_64/usd/usdGeom/resources/generatedSchema.usda). And we also have UsdShadeTokens.cs and UsdSkelTokens.cs to store other schema defined tokens. 

If we want static Tokens for our own needs we should add them in a different file.